src/features/query-bar/__tests__/__snapshots__/FilterButton-test.js.snap

```diff
@@ -21,7 +21,7 @@ exports[`feature/query-bar/components/filter/FilterButton render should render F
 >
   <Button
     className="query-bar-button"
-    isDisabled={true}
+    isDisabled={false}
```

Comment:
Expected?

Should the test description be rephrased to "should enable FilterButton when columns is non-empty"?